src/output/icons.rs

```diff
@@ -139,11 +148,15 @@ const DIRECTORY_ICONS: Map<&'static str, char> = phf_map! {
     "node_modules"        => Icons::FOLDER_NPM,     // 
     "npm_cache"           => Icons::FOLDER_NPM,     // 
     "pam.d"               => Icons::FOLDER_KEY,     // 󰢬
+    "portage"             => '\u{f30d}',            // 
     "Pictures"            => '\u{f024f}',           // 󰉏
+    "root"                => '\u{f0250}',           // 󰉐
+    "src"                 => '\u{f19fc}',           // 󱧼
```

Comment:
Not sure why source code would be a wrench folder.

Reply:
I guess I was thinking wrench -> build -> source code. If you don't like it I can change it, or we can go with a different icon.